src/js/views/globals/search/patient-search_views.js

```diff
@@ -99,6 +105,11 @@ const PicklistItem = View.extend({
           {{#if identifiers.0.value}}{{matchText identifiers.0.value search}}{{else}}&ndash;{{/if}}
         </div>
       {{/if}}
+      {{#if hasFieldQueryResults}}
+        <div class="patient-search__picklist-item-identifier u-text--overflow">
```

Comment:
I'd like to use this same class for both the identifier and field query column. But maybe it should be renamed to something generic.